### PR TITLE
Media release template

### PR DESCRIPTION
**Pull request by sukhrajghuman**
Draft media release template.

### CODE REVIEW

**Comment by sukhrajghuman** [comment]
Should we use lorem ipsum or an actual example? @gordongrace

**Comment by gordongrace** [comment]
We've been using a combination of lorem ipsum and near-real examples elsewhere in the documentation site. Perhaps a national, timeless media release would serve as a good example:

http://media.bom.gov.au/releases/675/aurora-activity-may-light-up-skies-in-southern-australia-this-week/

File: docs/media-release/index.html [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+<!--[if !(IE)]><!-->
+	<html class="no-js" lang="en">
+<!--<![endif]-->
+
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width">
+
+	<title>Australian Government Design System | Form</title>
+
```
Comment (gordongrace) [inline docs/media-release/index.html]: Add AGLS/OG/Twitter metadata for a media release?

File: docs/media-release/index.html [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+						<div class="au-main-nav__overlay" aria-controls="main-nav-default-dark" onClick="return AU.mainNav.Toggle( this )"></div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</nav>
+
+	<!-- MAIN BODY -->
+	<main id="content">
+		<!--CONTENT-->
```
Comment (gordongrace) [inline docs/media-release/index.html]: No breadcrumbs?

File: docs/media-release/index.html [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+		<div class="au-body container-fluid">
+			<div class="row">
+			<article class="col-md-7 media-release">
+				<h1>Third release of the Trusted Digital Identity Framework</h1>
+				<p class="media-release__caption">
+					We’ve released the third part of the Trusted Digital Identity Framework (TDIF). TDIF sits across all elements of the digital identity program to ensure a consistent approach to usability, accessibility, privacy protection, security and more.
+				</p>
+				<div class="media-release__metadata">
```
Comment (gordongrace) [inline docs/media-release/index.html]: This markup pattern seems better suited to a description list.

File: docs/media-release/index.html [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+						<li>privacy advocates</li>
+						<li>digital identity experts</li>
+						<li>the financial sector</li>
+						<li>industry groups</li>
+						<li>standards bodies</li>
+						<li>vendors</li>
+						<li>users and members of the public</li>
+					</ul>
+					<p>Following the consultation period in January, we’re releasing part 3 of the TDIF, which includes 3 new documents and updates to 9 existing documents.</p>
+					<figure class="media-release__figure">
```
Comment (gordongrace) [inline docs/media-release/index.html]: Do we have other examples where we're using the `figure` element?

File: docs/media-release/index.html [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+						<li>updates to the accreditation process to include the ongoing accreditation obligations for TDIF participants. This includes the annual assessment and the conditions when re-accreditation will be required.</li>
+						<li>updates to the protective security requirements to ensure they align with the current edition (2018) of the Australian Government Protective Security Policy Framework and Information Security Manual.</li>
+						<li>updates to the attribute profile to allow the sharing of additional identity attributes under certain situations (for example, to investigate instances of fraud).</li>
+					</ul>
+					<p>As the program develops and we learn more about our users, we will continue to iterate the TDIF. The next release will focus on authorisations and the needs of people who act on behalf of others when interacting with government.</p>
+					<p>You can read more about how we’re developing the TDIF in a blog post from the director of digital identity policy, Shannon Peterson.</p>
+				</div>
+
+				<div class="au-callout">
```
Comment (gordongrace) [inline docs/media-release/index.html]: Good choice.

File: docs/media-release/index.html [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+	</nav>
+
+	<!-- MAIN BODY -->
+	<main id="content">
+		<!--CONTENT-->
+		<div class="au-body container-fluid">
+			<div class="row">
+			<article class="col-md-7 media-release">
+				<h1>Third release of the Trusted Digital Identity Framework</h1>
+				<p class="media-release__caption">
```
Comment (gordongrace) [inline docs/media-release/index.html]: Use of the `au-display-*` classes seems more natural here, but brings additional visual weight. 

Could we explore a `au-paragraph-introduction` option to keep it generic?

File: src/sass/main.scss [hunk 7]
```diff
@@ -88,3 +88,30 @@ form {
...
+hr {
+	border-color: lightgray !important;
+}
+
+.media-release__caption {
+	color: $AU-color-foreground-muted;
+	@include AU-fontgrid('md');
+}
+
+.media-release__metadata {
```
Comment (gordongrace) [inline src/sass/main.scss]: Metadata blocks would be common to multiple content types. Is a `media-release__` prefix required?

File: docs/media-release/index.html (hunk 2 above)
Comment (sukhrajghuman) [reply docs/media-release/index.html]: good idea

File: docs/media-release/index.html (hunk 1 above)
Comment (sukhrajghuman) [reply docs/media-release/index.html]: sure

File: src/sass/main.scss (hunk 7 above)
Comment (sukhrajghuman) [reply src/sass/main.scss]: this is called [BEM](http://getbem.com/naming/)

File: docs/media-release/index.html (hunk 6 above)
Comment (sukhrajghuman) [reply docs/media-release/index.html]: `au-display--` doesn't apply the appropriate color shading. I've used the AU-fontgrid() mixin instead to adjust the size.

This is using [BEM](http://getbem.com/naming/)

We can rename it to `media-release__introduction`

File: docs/media-release/index.html (hunk 4 above)
Comment (sukhrajghuman) [reply docs/media-release/index.html]: nope, just thought I'd try it out here

File: docs/media-release/index.html (hunk 3 above)
Comment (sukhrajghuman) [reply docs/media-release/index.html]: 👍

**Comment by sukhrajghuman** [comment]
Closing this for now.
It was too similar to the content page template. With the only major difference being the content itself. That begs the question should we dictate content, or show common patterns and how they should be implemented.

**Pull request closed without merging**